ocean_lib/ocean/ocean_assets.py

```diff
@@ -519,7 +516,7 @@ def download_asset(
 
         return download_asset_files(
             asset=asset,
-            provider_uri=provider_uri,
+            provider_uri=self._config.provider_url,
```

Comment:
nope, no need for provider_uri here, either. Inside download_asset files, take the access service.